src/App.css

```diff
@@ -31,6 +31,11 @@ html {
     width: 8px;
     height: 3px;
 }
+
+::-webkit-scrollbar-button {
+    display: none; /* This will hide the scrollbar buttons */
+}
+
 ::-webkit-scrollbar-button {
```

Comment:
you have two selectors that are the same, so if you display:none; you can remove the next selector which indicates the background-color.

Reply:
Okay.